python_src/src/lamp_py/ingestion/ingest.py

```diff
@@ -72,7 +73,7 @@ def ingest_files(files: List[str], metadata_queue: Queue) -> None:
     # GTFS Static Schedule must be processed first for performance manager to
     # work as expected
     if ConfigType.SCHEDULE in converters:
-        converters[ConfigType.SCHEDULE].convert()
+        moved_static_files = converters[ConfigType.SCHEDULE].convert()
```

Comment:
get the static files that were moved. they'll be added to the moved realtime files.